### PR TITLE
Run perltidy on the dist

### PR DESCRIPTION
**Pull request by simbabque**
:broom:

### CODE REVIEW

File: dist.ini [hunk 1]
```diff
@@ -65,10 +65,10 @@ code = s{(^\s+or diag 'got warnings: ', explain\(\\\@warnings\);)}{$1\n\}};
 [Test::Portability]
 ; TODO perltidy for NoTabs and namespace::autoclean
 ; [Test::CleanNamespaces] ; TODO
-; [Test::NoTabs] ; TODO
+[Test::NoTabs] ; TODO
```
Comment (oalders) [inline dist.ini:68]: I think we can remove the `TODO` label?

File: t/form.t [hunk 2]
```diff
@@ -543,24 +543,24 @@ $f = HTML::Form->parse( <<EOT, "http://www.example.com" );
   <table>
     <TR>
       <TD align="left" colspan="2">
-	&nbsp;&nbsp;&nbsp;&nbsp;Keep me informed on the progress of this election
-	<INPUT type="checkbox" id="keep_informed" name="keep_informed" value="yes" checked>
+  &nbsp;&nbsp;&nbsp;&nbsp;Keep me informed on the progress of this election
+  <INPUT type="checkbox" id="keep_informed" name="keep_informed" value="yes" checked>
       </TD>
```
Comment (oalders) [inline t/form.t:548]: Not sure that it matters, but the alignment got messed up here. I'm guess it was tabs before?

**Review by oalders — CHANGES_REQUESTED** [review]
Looks great. Thanks for doing this. Can we squash it down?

File: t/form.t (hunk 2 above)
Comment (simbabque) [reply t/form.t:548]: Yes, it was. That was the only thing that failed the tabs test. I converted it, and it looked like it didn't change the visual indentation, but I guess I was wrong. Thanks for pointing it out.

**Comment by simbabque** [comment]
> Looks great. Thanks for doing this. Can we squash it down?

Should all be fixed now. Thanks for the review. Go ahead and squash it. :)

**Pull request merged**